### PR TITLE
Add .test target for packages

### PR DESCRIPTION
**Pull request by andralex**

### CODE REVIEW

File: posix.mak [hunk 1]
```diff
@@ -380,10 +380,16 @@ moduleName=$(subst /,.,$(1))
 unittest/%.run : $(ROOT)/unittest/test_runner
 	$(QUIET)$(RUN) $< $(call moduleName,$*)
 
-# target for quickly running a single unittest (using static phobos library)
+# Target for quickly running a single unittest (using static phobos library).
+# For example: "make std/algorithm/mutation.test"
 %.test : %.d $(LIB)
 	$(DMD) $(DFLAGS) -main -unittest $(LIB) -defaultlib= -debuglib= -L-lcurl -run $<
```
Comment (WalterBright) [inline posix.mak]: add ```-cov``` flag

**Comment by CyberShadow** [comment]
Auto-merge toggled on

**Comment by andralex** [comment]
Auto-merge toggled off

**Comment by andralex** [comment]
lemme investigate dat cov

**Comment by andralex** [comment]
added -cov

**Comment by WalterBright** [comment]
Auto-merge toggled on

**Comment by jacob-carlborg** [comment]
I would have expected `make test std/algorithm/mutation.d`. Seems like this would work better with tab completion.

**Pull request merged**